### PR TITLE
Feature: Added support for changing groups direction

### LINKED ISSUE #8583: Feature: Groups should follow ascending/descending preference
### Requirements 
- Groups should follow ascending/descending preference

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #8583

**Validation**
How did you test these changes?
- [X] Built and ran the app
- [X] Tested the changes for accessibility

**Details**
Sorting and groups can now be selected in ascending or descending, respectively.

**Screenshots**
image

### CODE REVIEW

File: src/Files.App/ServicesImplementation/DateTimeFormatter/AbstractDateTimeFormatter.cs [hunk 1]
```diff
@@ -26,14 +26,14 @@ public ITimeSpanLabel ToTimeSpanLabel(DateTimeOffset offset)
...
-				_ when now.Year == time.Year => new Label("ItemTimeText_ThisYear", "\ue163", 5),
-				_ => new Label("ItemTimeText_Older", "\uEC92", 6),
+				_ when now.Date == time.Date => new Label("Today", "\ue184", 7),
+				_ when y.Date == time.Date => new Label("ItemTimeText_Yesterday", "\ue161", 6),
+				_ when diff.Days < 7 && w.Year == time.Year && GetWeekOfYear(w) == GetWeekOfYear(time) => new Label("ItemTimeText_ThisWeek", "\uE162", 5),
+				_ when diff.Days < 14 && w.Year == time.Year && GetWeekOfYear(w) == GetWeekOfYear(time) => new Label("ItemTimeText_LastWeek", "\uE162", 4),
+				_ when now.Year == time.Year && now.Month == time.Month => new Label("ItemTimeText_ThisMonth", "\ue163", 3),
+				_ when now.AddMonths(-1).Year == time.Year && now.AddMonths(-1).Month == time.Month => new Label("ItemTimeText_LastMonth", "\ue163", 2),
+				_ when now.Year == time.Year => new Label("ItemTimeText_ThisYear", "\ue163", 1),
+				_ => new Label("ItemTimeText_Older", "\uEC92", 0),
```
Comment (yaira2) [inline src/Files.App/ServicesImplementation/DateTimeFormatter/AbstractDateTimeFormatter.cs:36]: Why this change?

File: src/Files.App/ViewModels/FolderSettingsViewModel.cs [hunk 2]
```diff
@@ -376,7 +392,7 @@ private static LayoutPreferences GetDefaultLayoutPreferences(string folderPath)
 
 			if (folderPath == CommonPaths.DownloadsPath)
 				// Default for downloads folder is to group by date created
-				return new LayoutPreferences() { DirectoryGroupOption = GroupOption.DateCreated };
+				return new LayoutPreferences() { DirectoryGroupOption = GroupOption.DateCreated, DirectoryGroupDirection = SortDirection.Descending };
```
Comment (yaira2) [inline src/Files.App/ViewModels/FolderSettingsViewModel.cs:395]: Can you split this on to separate lines?

File: src/Files.App/ServicesImplementation/DateTimeFormatter/AbstractDateTimeFormatter.cs (hunk 1 above)
Comment (hishitetsu) [reply src/Files.App/ServicesImplementation/DateTimeFormatter/AbstractDateTimeFormatter.cs:36]: This is to ensure that the dates are ordered from oldest to newest when the order direction is ascending.

File: src/Files.App/ViewModels/FolderSettingsViewModel.cs (hunk 2 above)
Comment (hishitetsu) [reply src/Files.App/ViewModels/FolderSettingsViewModel.cs]: Done.

**Comment by yaira2** [comment]
Can you apply the same to the right click menu in the folder layouts?

**Comment by hishitetsu** [comment]
I see, I'll try it later.

**Comment by hishitetsu** [comment]
Done.

**Review by yaira2 — APPROVED** [review]
LGTM
